modules/light-clients/14-avalanche/client_state.go

```diff
@@ -181,33 +192,81 @@ func verifyDelayPeriodPassed(ctx sdk.Context, store sdk.KVStore, proofHeight exp
 
 	return nil
 }
-func (cs *ClientState) CheckForMisbehaviour(ctx sdk.Context, cdc codec.BinaryCodec, clientStore sdk.KVStore, msg exported.ClientMessage) bool {
-	//TODO implement me
-	panic("implement me")
-}
 
-func (cs *ClientState) VerifyNonMembership(ctx sdk.Context, clientStore sdk.KVStore, cdc codec.BinaryCodec, height exported.Height, delayTimePeriod uint64, delayBlockPeriod uint64, proof []byte, path exported.Path) error {
-	//TODO implement me
-	panic("implement me")
-}
+func (cs *ClientState) UpdateStateOnMisbehaviour(ctx sdk.Context, cdc codec.BinaryCodec, clientStore sdk.KVStore, _ exported.ClientMessage) {
```

Comment:
Suggest moving this to update.go to be consistent with the tendermint light client. Same with the UpdateState function.

Reply:
update

Reply:
update